src/routes/jira/sync/jira-sync-post.test.ts

```diff
@@ -40,20 +62,38 @@ describe("sync", () => {
 		}, installation.sharedSecret);
 	});
 
-	it("should return 200 on correct post for /jira/sync", async () => {
+	it("should return 200 on correct post for /jira/sync for Cloud app", async () => {
 		return supertest(app)
 			.post("/jira/sync")
 			.query({
 				jwt
 			})
 			.send({
-				installationId: installation.id,
+				installationId: installationIdForCloud,
 				jiraHost,
 				syncType: "full"
 			})
 			.expect(202)
 			.then(() => {
-				expect(sqsQueues.backfill.sendMessage).toBeCalledWith(expect.objectContaining({ installationId: installation.id, jiraHost, startTime: expect.anything() }), expect.anything(), expect.anything());
+				expect(sqsQueues.backfill.sendMessage).toBeCalledWith(expect.objectContaining({ installationId: installationIdForCloud, jiraHost, startTime: expect.anything() }), expect.anything(), expect.anything());
```

Comment:
can you also expect the `gitHubAppConfig` in the `sendMessagePayload`?

Reply:
Updated